src/seml/database.py

```diff
@@ -258,7 +258,7 @@ def clean_unreferenced_artifacts(db_collection_name=None, yes=False):
     collection_names = collection_names - collection_blacklist
 
     referenced_files = set()
-    tq = tqdm(collection_names)
+    tq = track(collection_names)
     logging.info('Scanning collections for orphaned artifacts...')
     for collection_name in tq:
         tq.set_postfix(collection=collection_name)
```

Comment:
tq is an iterable of the sequence values.
The method `set_postfix` unlike tqdm, doesn't exist here

Reply:
Thanks a lot for spotting this! I fixed it in fa1a10606c2930e21244bbe982f2604f209e5718